ert3/workspace/_workspace.py

```diff
@@ -31,7 +31,7 @@ def experiment_have_run(workspace_root, experiment_name):
 def initialize(path):
     path = Path(path)
     if load(path) is not None:
-        sys.exit("Already inside an ERT workspace")
+        raise ValueError("Already inside an ERT workspace.")
```

Comment:
This is a good change, as only `main` should raise `sys.exit`. You are free to make up your own exception though, as different errors are applicable to different things. We can then catch the error in `main` and exit with a nice message.

Reply:
This should definitively not be a `sys.exit`. Instead, I suggest we raise an `IllegalWorkspaceOperation` that we can catch in the console and turn into a `sys.exit` with the exception message...